backend/app/Models/BattleGroupCard.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class BattleGroupCard extends Model
+{
+    public function card()
```

Comment:
Just because you need a one-to-many relationship between BattleGroups and cards doesn't mean you need a model, right? Can't you just have a pivot table and have BattleGroups->hasMany(Card, ..., ..., ...)

Reply:
`BattleGroup` and` Card` are many2many, (via `BattleGroupCard` right now); 

I _could_ do it with a hasManyThrough but that gets a little messy/verbose when there's data being stored in the pivot table `sort_order` on `battle_group_cards` since you have to do the `withPivot` 🤷‍♂️ 

overall it's functionally the same